platform/logging/appenders/console/ConsoleAppender.ts

```diff
@@ -0,0 +1,14 @@
+import { LogRecord } from '../../LogRecord';
+import { BaseAppender } from '../base/BaseAppender';
+import { ConsoleAppenderConfig } from './ConsoleAppenderConfig';
+
+export class ConsoleAppender extends BaseAppender {
+  constructor(protected readonly config: ConsoleAppenderConfig) {
+    super(config)
+  }
+
+  append(record: LogRecord) {
+    super.append(record);
+    console.log(this.logRecordToFormattedString(record));
```

Comment:
Here is how it can be done, every appender can specifically post/pre-format any log message part. ConsoleLogger applies color using this extension hook. Though not sure about different colors for different context like it's done currently, do you think we need to support it?